classes/module/Module.php

```diff
@@ -473,6 +486,18 @@ public function install()
 
         // Adding Restrictions for client groups
         Group::addRestrictionsForModule($this->id, Shop::getShops(true, null, true));
+        PrestaShopLogger::addLog(
+            Context::getContext()->getTranslator()->trans(
+                'Module installed successfully: %s v%s',
+                [$this->name, $this->version],
+                'Admin.Modules.Notification'
+            ),
+            1,
+            null,
+            'Module',
+            null,
```

Comment:
Why not including the module ID here for the actions where it's available?

Reply:
Because the id is not a relevant information : it change at every install/unintall. Module name is a better identifier.